apm-lambda-extension/extension/http_server_test.go

```diff
@@ -46,15 +47,17 @@ func TestInfoProxy(t *testing.T) {
 		apmServerUrl:               apmServer.URL,
 		apmServerSecretToken:       "foo",
 		apmServerApiKey:            "bar",
-		dataReceiverServerPort:     "127.0.0.1:1234",
+		dataReceiverServerPort:     ":1234",
```

Comment:
Not mandatory, but since we're changing the semantics of this variable/property to include _only_ the port instead of the server and the port, do we want to change its name to `dataReceiverPort`?

Reply:
I think that makes the most sense. I don't know if users even have the option of starting a server on another host other than localhost from within a lambda environment? I'll open another PR to change this so we can have the discussion there.